actions/setup-gap/action.yml

```
@@ -190,23 +208,32 @@ runs:
       if: inputs.use-k8s == 'true'
       shell: sh
       env:
+        DYNAMIC_PROXY_PORT: ${{ inputs.dynamic-proxy-port }}
         ENABLE_PROXY_DEBUG: ${{ inputs.enable-proxy-debug }}
-        ENVOY_LOG_LEVEL: "info"
+        GITHUB_OIDC_TOKEN_HEDER_NAME:
```

```suggestion
        GITHUB_OIDC_TOKEN_HEADER_NAME:
```

Would also need to update all refs to this new name.